pilot/pkg/config/kube/gateway/testdata/delegated.yaml.golden

```diff
@@ -4,7 +4,6 @@ metadata:
   annotations:
     internal.istio.io/gateway-semantics: gateway
     internal.istio.io/gateway-service: istio-ingressgateway.istio-system.svc.domain.suffix
-    internal.istio.io/parents: Gateway/gateway/apple.istio-system
```

Comment:
Did we replace this with some other indication of what generated this internal object?